docs/source/quickstart.mdx

```diff
@@ -1,296 +1,550 @@
+### Understanding Tools
 
-### Additional Resources
+MCP tools are GraphQL operations that Claude can execute. Each operation in your `operations/` directory becomes a tool with:
 
-Check out these blog posts to learn more about Apollo MCP Server:
-- [Getting started with Apollo MCP Server](https://www.apollographql.com/blog/getting-started-with-apollo-mcp-server-for-any-graphql-api)
-- [The Future of MCP is GraphQL](https://www.apollographql.com/blog/the-future-of-mcp-is-graphql)
+- **Tool name**: Derived from the operation name
+- **Description**: From GraphQL comments or operation descriptions
+- **Parameters**: Mapped from GraphQL variables
+- **Response**: The GraphQL response data
 
-### Advanced Options
+### Defining Custom Tools
 
-<details>
-<summary>Alternative ways to run the MCP Server</summary>
-
-#### Using STDIO Transport
-
-You can run the MCP Server using STDIO transport instead of Streamable HTTP. This is useful for certain environments or when you need more direct control over the server process.
-
-1. Download the binary of the latest version of Apollo MCP Server
-2. Use MCP Inspector to run the server:
-
-    ```yaml title="Config for stdio transport"
-    endpoint: https://thespacedevs-production.up.railway.app/
-    operations:
-      source: local
-      paths:
-      - <absolute-path-to-MCP-example-dir>/operations
-    schema:
-      source: local
-      path: <absolute-path-to-MCP-example-dir>/api.graphql
-    transport:
-      type: stdio
-    ```
+Create new tools by adding GraphQL operation files to your `operations/` directory:
 
-    ```sh
-    npx @modelcontextprotocol/inspector apollo-mcp-server <path to the preceding config>
-    ```
+```graphql
+# operations/GetProducts.graphql
 
-3. Configure Claude Desktop to use STDIO:
+# Retrieves product information by ID
+query GetProducts($productId: ID!) {
+  product(id: $productId) {
+    id
+    name
+    price
+    description
+  }
+}
+```
 
-    ```json
-    {
-      "mcpServers": {
-        "thespacedevs": {
-          "command": "<absolute-path-to-MCP-server-binary>",
-          "args": [
-            "<path to the preceding config>"
-          ]
-        }
-      }
+Apollo MCP Server automatically detects new `.graphql` files and exposes them as tools. No server restart needed!
+
+**Learn more:** [Defining MCP Tools](https://www.apollographql.com/docs/apollo-mcp-server/define-tools)
+
+### Testing Operations in GraphOS Studio
+
+The `rover dev` command includes a built-in GraphQL explorer:
+
+1. Open http://localhost:4000 in your browser
+2. You'll see GraphOS Studio's Sandbox Explorer
+3. Test your operations directly:
+   - Browse your schema
+   - Run queries and mutations
+   - Experiment with variables
+   - View responses
+
+This is invaluable for:
+- Verifying operations before exposing them as MCP tools
+- Understanding your schema structure
+- Debugging query issues
+- Exploring available data
+
+### Testing with MCP Inspector
```

Comment:
We already have a section about this above 🤔

Reply:
I would remove the inspector part from step 4